docs/index.rst

```diff
@@ -84,6 +84,7 @@ Content
     concepts
     scheduler
     executor/index
+    dag-run
```

Comment:
I think this probably makes more sense as a page under Concepts -- it doesn't fit with the rest of the top level items we have.

WDYT?

Reply:
@mik-laj and I were having a discussion on the same perspective that the concepts page needs to be broken down and some subpages need to move inside it.  Since that would require changes in many pages, can we have that as part of another PR?

Reply:
We can have a subpage like https://airflow.readthedocs.io/en/stable/howto/index.html and have different topics there

Reply:
@KKcorps What did we decide about this? Merge this PR as is then you split up concepts in to multiple pages afterwards?

Reply:
Yes, splitting up the page is a big effort. I think we should merge this and that split it up later.